src/libs/ModifiedExpenseMessage.ts

```diff
@@ -126,6 +135,20 @@ function getForDistanceRequest(newMerchant: string, oldMerchant: string, newAmou
     });
 }
 
+function getForExpenseMovedFromSelfDM(destinationReportID: string) {
+    const destinationReport = allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${destinationReportID}`];
+    const rootParentReport = getRootParentReport(destinationReport);
+
+    // The "Move report" flow only supports moving expenses to a policy expense chat or a 1:1 DM.
+    const reportName = isPolicyExpenseChat(rootParentReport) ? getPolicyExpenseChatName(rootParentReport) : buildReportNameFromParticipantNames({report: rootParentReport});
+    const policyName = getPolicyName(rootParentReport, true);
```

Comment:
> I have an issue with the dependency cycle https://github.com/Expensify/App/pull/51517#issuecomment-2519967159

Can't we move this part to ReportUtils to avoid dependency cycle issue?